### DISCUSSION TITLE
madaidan's problematic software recommendations.

### DISCUSSION THREAD

**Discussion by amano-kenji** [netblue30/firejail#5435] [General]
* https://madaidans-insecurities.github.io/security-privacy-advice.html recommends windows and MacOS and tells people to avoid linux. However, windows is a closed-source spyware and has backdoors for microsoft and governments. MacOS also is a closed-source spyware with backdoors and regularly reports user activities to Apple which will (be forced to?) sell user data to governments.
  * Lesson // The highest security doesn't really matter for spywares with backdoors.
* https://madaidans-insecurities.github.io/vpns.html seems to discourage use of VPN. Although his criticism on VPNs is correct, using VPN is still better than not using VPN. If you don't use VPN, your ISP will just sell your browsing data to governments. Using VPN provides an extra protection against mass surveillance, but be aware that VPNs can succumb to government pressure. The fact that many governments are trying to ban VPN should tell you that good VPNs actually try to protect user privacy. Good VPNs have done external audits or fought governments in legal courts. I use VPN "and" torbrowser. If you want to avoid censorship, VPN is perfectly good. Government censorship alone justifies VPN usage. Torbrowser is too slow for regular browsing. Also, torrenting without VPN will allow copyright trolls to try to extort money from you.
* https://madaidans-insecurities.github.io/firefox-chromium.html tells people to avoid firefox and recommends chromium. However, chromium, by default, is somewhat a spyware. He didn't even mention brave which is a much more private version of chromium. I use multiple browsers to implement browser isolation. I use google chrome for google services, brave for local services, hardened firefox for general browsing, and so on. Browser isolation is better than using one browser for everything.
* https://madaidans-insecurities.github.io/messengers.html recommends Signal which requires a phone number. While Signal is okay when you are using a smartphone, the phone number requirement is bad. XMPP with OMEMO encryption doesn't require a phone number.
* https://madaidans-insecurities.github.io/browser-tracking.html doesn't point out that hardened firefox on VPN provides higher baseline levels of privacy than plain firefox without VPN. I know that fingerprinting resistance won't magically prevent all forms of browser fingerprinting, but it is much better than nothning. That's why I use hardened firefox along with other web browsers including torbrowser.
* https://madaidans-insecurities.github.io/linux.html tells people to avoid firejail and use bubblewrap. The problem is that bubblewrap is not intended to be used directly. It was designed to be used by higher-level sandboxing programs like flatpak. If there is something like firejail that uses bubblewrap, then I may use it.

There are a few possibilities.

* Madaidan may have computer knowledge but is not that bright in the sense that he doesn't account for worldwide tyranny that's growing rapidly. In a free world, privacy may not be very important. But, we live in illusion of freedom erected by covert forms of tyranny.
* Or, he knowingly recommends spywares with backdoors in order to screw gullible people.

Either way, computer knowledge by itself is not very useful without knowledge about the world we live in. Computer knowledge doesn't have value if it is not applied to protect moral people who speak truths despite threats of violence (from governments).

**Comment by kmk3** [comment]
Relates to:

* <https://github.com/netblue30/firejail/discussions/4601>

**Reply by kmk3** [reply]
@rusty-snake Thoughts on adding this discussion to the list on #4601?
